### DISCUSSION TITLE
ERROR LDAP SYNC AFTER CHANGE PERIODIC FULL SYNC

### DISCUSSION THREAD

**Discussion by hakimnorizman-work** [keycloak/keycloak#23299] [Q&A]
Hi, basically I just enabled the periodic full sync by following a website for the settings and I got this error 
<img width="355" alt="image" src="https://github.com/keycloak/keycloak/assets/114655017/53f63515-9f42-447b-99a7-ee3357a6882e">

Now I am unable to change it as it always prompt this error whenever I clicked on LDAP federation

**Comment by jonkoops** [comment]
If this is a reproducible problem in the latest version of Keycloak please open an issue to report this as a bug.

**Reply by hakimnorizman-work** [reply]
Hi, 
I will try to upgrade to latest version first and see if the issue occurs again.
